nested/storageAccount.json

```diff
@@ -12,7 +12,7 @@
     "resources": [
         {
             "type": "Microsoft.Storage/storageAccounts",
-            "apiVersion": "2017-06-01",
+            "apiVersion": "latest",
```

Comment:
Let's not use "latest" for apiVersion, let's have a unique date string around this commit/time-frame for every applicable mention of apiVersion.